src/particles/ImpactXParticleContainer.H

```diff
@@ -107,6 +107,8 @@ namespace impactx
         void
         SetRefParticle (RefPart const refpart);
```

Comment:
Can you please add a Doxygen doc string to these two functions?

Does not need to be long, but its good to add one since we use abbreviations in the method names.

Reply:
Ok, will do.